core/modules/system/css/messages.theme.css

```diff
@@ -56,50 +56,50 @@
   margin-left: 0;
 }
 
-.status {
+.messages.status {
   color: #234600;
   background-color: #e9eebc;
 }
-.status a {
+.messages.status a {
   color: #234600;
 }
-div.status:before {
+div.messages.status:before {
   background-image: url(../../../misc/message-24-ok.png);
   background-color: #cfde56;
 }
 
-.warning {
+.messages.warning {
   color: #525252;
   background-color: #faf5c9;
 }
-.warning a {
+.messages.warning a {
   color: #525252;
 }
-div.warning:before {
+div.messages.warning:before {
```

Comment:
Don't know why the `:before` rules need a `div` tag in selector... probably some cascading reason.

Reply:
Don't know either @opi. I imagine specificity wars at some point. I chose to keep what we already had in place. so not to break things.

Reply:
agreed